.gitignore

```diff
@@ -41,6 +41,8 @@ server/
 *.db
 python-*-docs-html
 /.cask/
+geben/
```

Comment:
We probably want to set `geben-temporary-file-directory` to a location inside of the cache directory (`spacemacs-cache-directory`), instead of having it directly inside of `.emacs.d`.

In any case, we need to set `geben-temporary-file-directory` because the default value is hard-coded to use `"~/.emacs.d"` and not `user-emacs-directory`, limiting portability.